frontend/src/pages/org/workflow-editor.ts

```diff
@@ -2110,28 +2112,23 @@ https://archiveweb.page/images/${"logo.svg"}`}
       : [];
     const primarySeed: Seed = {
       url: primarySeedUrl,
-      scopeType: this.formState.scopeType,
+      scopeType: this.formState.scopeType === "custom" ? "prefix" : this.formState.scopeType,
```

Comment:
This makes sense in the context of this PR and its description but might be confusing to someone coming across it later - might be a good place to leave a comment in the code?

Reply:
Added comments about this!